charts/shoot-core/components/charts/vpn-shoot/templates/secret.yaml

```diff
@@ -2,6 +2,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
+  #doubt
   name: {{ include "vpn-shoot.certs.name" . }}
```

Comment:
same doubt as above.

Reply:
Yes, let's do this together in a call next week.